ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/validation/DataColumnSidecarGossipValidator.java

```diff
@@ -348,8 +365,16 @@ private boolean verifyDataColumnSidecarInclusionProof(final DataColumnSidecar da
             dataColumnSidecar.getBlockBodyRoot()))) {
       return true;
     }
-
-    return miscHelpersFulu.verifyDataColumnSidecarInclusionProof(dataColumnSidecar);
+    try (MetricsHistogram.Timer ignored =
+        dataColumnSidecarInclusionProofVerificationTimeSeconds.startTimer()) {
+      return miscHelpersFulu.verifyDataColumnSidecarInclusionProof(dataColumnSidecar);
+    } catch (final Throwable t) {
+      LOG.error(
```

Comment:
I think it would be ok to keep it in debug. Also message looks incorrect, something like this could be more appropriate: 
```java
      LOG.debug(
          "Failed to verify inclusion proof of data column sidecar {}",
          dataColumnSidecar.toLogString(),
          t);
```